src/app/shared/doc-viewer/docs-api-module-import/docs-api-module-import.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { CopierService } from '../../copier/copier.service';
+
+const EXPORT_NAME_PLACEHOLDER = '{{exportName}}';
+const MODULE_IMPORT_PATH_PLACEHOLDER = '{{moduleImportPath}}';
+const MODULE_IMPORT_TEMPLATE = `import {${EXPORT_NAME_PLACEHOLDER}} from '${MODULE_IMPORT_PATH_PLACEHOLDER}';`;
+
+@Component({
+  selector: '[docs-api-module-import]',
+  template: `
+    <p class="docs-api-module-import">
+      <code>
+        {{ moduleImport }}
+      </code>
+    </p>
+    <button
+      class="docs-api-module-import-copy"
+      aria-label="Copy module import"
+      mat-icon-button
+      (click)="copyModuleImport()"
+      matTooltip="Copy module import">
+      <span class="material-icons">file_copy</span>
```

Comment:
```suggestion
      <span class="material-icons">content_copy</span>
```
This is the icon that angular.io uses.